Bugzilla/Elastic/Indexer.pm

```diff
@@ -213,7 +155,8 @@ sub _select_updated_ids {
 sub bulk_load_ids {
     my ($self, $class, $ids) = @_;
 
-    $self->put_mapping($class);
+    $self->_create_index($class);
```

Comment:
Because bulk_load_ids() is public, it could be called when the index doesn't exist.